seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -686,7 +686,7 @@
         "Agora, vamos importar e aplicar um estilo de fonte por meio do Google Fonts.",
         "Primeiro, faça um <code>chamado</code> ao Google Fonts para poder utilizar a fonte chamada <code>Lobster</code> e carregá-la em seu HTML.",
         "Para fazer isso, copie o código abaixo e insira-o na parte superior de seu editor de texto:",
-        "<code>&#60;link href=\"http://fonts.googleapis.com/css?family=Lobster\" rel=\"stylesheet\" type=\"text/css\"&#62;</code>",
+        "<code>&#60;link href=\"https://fonts.googleapis.com/css?family=Lobster\" rel=\"stylesheet\" type=\"text/css\"&#62;</code>",
```

Comment:
Nope that's going to confuse people. Its better this way. But having said that this could be done in actual code logic for FCC but not in challenges.

Can be shocker to beginners.